tcms/testplans/templates/testplans/mutable.html

```diff
@@ -0,0 +1,96 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block head %}
+    {{ form.media }}
+{% endblock %}
+
+<head>
+    <title>{% block title %}{% trans "Create new TestPlan" %}{% endblock %}</title>
+</head>
+
+{% block contents %}
+
+    <div class="container-fluid container-cards-pf">
+        <form class="form-horizontal" action="{% url 'plans-new' %}" enctype="multipart/form-data" method="post">
+            {% csrf_token %}
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_name">{% trans "Name" %}</label>
+                <div class="col-md-11 col-lg-11">
+                    <input type="text" id="id_name" name="name" value="{% firstof form.name.value "" %}" class="form-control">
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="col-md-1 col-lg-1">
+                    <label for="id_product">{% trans "Product" %}</label>
+                    <a href="{% url 'admin:management_product_add' %}?_popup" id="add_id_product" alt="{% trans 'add new Product' %}" title="{% trans 'add new Product' %}">+</a>
```

Comment:
For the '+' sign to work you need the following (again hints from `testruns/mutable.html`):

For the on-load handler you have to add:
```
    $('#add_id_product').click(function() {
        return showRelatedObjectPopup(this);
    });

    $('#add_id_version').click(function() {
        return showRelatedObjectPopup(this);
    });
```

This requires the following resources inside the template (before files coming from our project):

```
<script src="{% static "grappelli/jquery/jquery.min.js" %}" type="text/javascript"></script>
<script src="{% static "grappelli/js/grappelli.min.js" %}" type="text/javascript"></script>
<script src="{% static "admin/js/admin/RelatedObjectLookups.js" %}" type="text/javascript"></script>
```

This will make the popup for adding new product work.